timescaledb/overview/release-notes/index.md

```diff
@@ -38,87 +38,88 @@ If you have questions about distributed hypertables, join our #multinode channel
-
-This new continuous aggregate format is faster than the previous version. 
-Because continuous aggregates no longer store partials, and re-aggregation is removed, 
-it is now possible to create indexes on continuous aggregates columns.
-This can further improve performance.
-There are no longer any restrictions on which types of aggregates you can use
-with continuous aggregates. All of these are now available:
-
-* aggregates with DISTINCT
-* aggregates with FILTER
-* aggregates with FILTER in HAVING clause
-* aggregates without combine function
-* ordered-set aggregates
-* hypothetical-set aggregates
-
-The existing continuous aggregates are safe to use and continue to work as before. 
-They are still supported, however they do not benefit from the performance advantages 
-of the new format.
-
-By default, TimescaleDB 2.7 and later uses the new continuous aggregate format.
-
-We recommend that you recreate your existing continuous aggregates to take advantage 
-of these improvements. To do this, you need the original data. A migration path for 
-migrating to the new continuous aggregate format without the original data is intended 
-in a future version of TimescaleDB.
-
-This release also adds a new optimization for queries with now() constraints.
-Queries on hypertables with many chunks and a `now()` constraints that would
-exclude all but the most recent chunk were dominated by planning time.
-This new optimization enhances the planner to take the `now()` constraint into
-consideration when planning thereby reducing the work that needs to be done
-in the planner.
-
-This release also contains bug fixes and minor feature improvements since the 2.7.0 release.
-
-<!-- <highlight type="note"> This release is low priority for upgrade. We recommend that you upgrade when you can. </highlight> -->
-
-<highlight type="important"> This release is medium priority for upgrade. We recommend that you upgrade at the next available opportunity. </highlight>
-
-<!-- <highlight type="warning"> This release is high priority for upgrade. We strongly recommend that you upgrade as soon as possible. </highlight> -->
-
-
-The experimental features in the 2.7 release are:
-
-* The `time_bucket_ng` function, a newer version of `time_bucket`. This function
-supports years, months, days, hours, minutes, seconds, and timezones.
-* `time_bucket_ng` support for N months and Timezones on continuous aggregates
-* APIs for chunk manipulation across data nodes in a distributed
-hypertable setup. This includes the ability to add a data node and move
-chunks to the new data node for cluster rebalancing.
+## Release notes
 
+## 2.8.0 (2022-08-30)
 
-We're committed to developing these experiments, giving the community a chance to
-provide early feedback and influence the direction of TimescaleDB's development.
- We'll travel faster with your input!
+This release adds major new features since the 2.7.2 release.
+We deem it moderate priority for upgrading.
```

Comment:
We used to have standard statements for this in the comments, so you could just uncomment the one you needed. What's happened to that?

Reply:
Sorry i'm not sure what you are referring to all this content is coming from https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md